homeassistant/components/automation/trace.py

```diff
@@ -2,148 +2,25 @@
 from __future__ import annotations
 
 from contextlib import contextmanager
-import datetime as dt
-from itertools import count
-from typing import Any, Deque
 
-from homeassistant.components.trace.const import DATA_TRACE, STORED_TRACES
-from homeassistant.components.trace.utils import LimitedSizeDict
-from homeassistant.core import Context
-from homeassistant.helpers.trace import TraceElement, trace_id_set
-from homeassistant.util import dt as dt_util
+from homeassistant.components.trace import AutomationTrace, async_store_trace
```

Comment:
Shouldn't `AutomationTrace` be renamed or, if it is automation specific, be in this file?

Reply:
Make up your mind: https://github.com/home-assistant/core/pull/48231/files#r600041503 🤣

Reply:
Fixed in #48288